invenio_communities/communities/records/api.py

```diff
@@ -50,5 +57,44 @@ class Community(Record):
         delete=False,
     )
 
+    dumper = ElasticsearchDumper(
+        extensions=[
+            RelationDumperExt('relations'),
+        ]
+    )
+
+    relations = RelationsField(
+        # FIXME: add identifiers to keys when merged in invenio-vocabularies
+        funding_award=PIDListRelation(
+            'metadata.funding',
+            relation_field='award',
+            keys=['number', 'title'],
+            pid_field=Award.pid,
+            cache_key='awards',
+        ),
+
+        funding_funder=PIDListRelation(
+            'metadata.funding',
+            relation_field='funder',
+            keys=['name'],
+            pid_field=Funder.pid,
+            cache_key='funders',
+        ),
+
+        organizations=PIDListRelation(
+            'metadata.organizations',
+            keys=['name'],
+            pid_field=Affiliation.pid,
+            cache_key='affiliations',
+        ),
+
+        types=PIDListRelation(
```

Comment:
major: `types` seems to be in this PR but there are no tests for it (just one), it's missing in the mappings 🤔